caching/base.py

```diff
@@ -8,7 +8,7 @@
 from django.db.models.sql import query, EmptyResultSet
 from django.utils import encoding
 
-from .compat import DEFAULT_TIMEOUT
+from .compat import DEFAULT_TIMEOUT, u
```

Comment:
The other way to deal with unicode literals is to do: `from __future__ import unicode_literals` and then remove them from the code. https://docs.djangoproject.com/en/1.8/topics/python3/#unicode-literals (Technically speaking, no change at all should be necessary since the unicode literals is only a problem on Python 3.0-3.2, which we're not targeting)